lib/scutil/host-fp-folding.c

```diff
@@ -55,12 +55,6 @@ fold_sanity_check(void)
   assert(sizeof(float128_t) == 16);
 }
 
-/*
- *  C99 feature: alert the compiler that the following code cares about
- *  the dynamic floating-point environment.
- */
-#pragma STDC FENV_ACCESS ON
```

Comment:
I understand that clang supports this pragma.  Is the pragma not important for the functionality of the code below?

Reply:
LLVM 12 ignores this pragma when running without strict FP support. See https://reviews.llvm.org/D90316 for information.

Reply:
It seems that `HasStrictFP` is `true` for X86 and PPC, but not for AArch64.

Reply:
Fixed.

Reply:
@shivaramaarao Please take a look at this again?